x-pack/solutions/observability/plugins/uptime/public/legacy_uptime/components/common/charts/ping_histogram.tsx

```diff
@@ -125,19 +125,17 @@ export const PingHistogramComponent: React.FC<PingHistogramComponentProps> = ({
     });
 
     content = (
-      <ChartWrapper
-        height={height}
-        loading={loading}
-        aria-label={i18n.translate('xpack.uptime.snapshotHistogram.description', {
-          defaultMessage:
-            'Bar Chart showing uptime status over time from {startTime} to {endTime}.',
-          values: {
-            startTime: moment(new Date(absoluteStartDate).valueOf()).fromNow(),
-            endTime: moment(new Date(absoluteEndDate).valueOf()).fromNow(),
-          },
-        })}
-      >
-        <Chart>
+      <ChartWrapper height={height} loading={loading}>
```

Comment:
This had to move lower because `ChartWrapper` assumes a role that does not permit aria-label attributes. The `Chart` element is the actual chart described by the label in any case, so this makes more sense.